sdk/src/utils/xmp_inmemory_utils.rs

```diff
@@ -23,6 +23,7 @@ use quick_xml::{
 use crate::{asset_io::CAIRead, jumbf_io::get_cailoader_handler, Error, Result};
 
 const RDF_DESCRIPTION: &[u8] = b"rdf:Description";
+const XMP_END: &[u8] = b"<?xpacket end=\"w\"?>";
```

Comment:
I guess you could go with the end=\'w\' single quote version since you said that is what the spec has.